misc/cleanup-dash.py

```diff
@@ -399,6 +399,11 @@ def add_annotation(dashboard):
     user_input = raw_input(prompt)
     if user_input:
         if user_input == 'Yes':
+            active_variables = [tag]
```

Comment:
Looks like we do not change `tag` here - can be a constant. Please use upper case for constants.